File: data/c++/pattern_4/code.cpp
#include <iostream>
using namespace std;

int main(void)
{
    int num;
    cout << "Enter the number of rows and columns for the square: ";
    cin >> num;

    for (int i = num; i >= 1; i--)
    {
        for (int j = 1; j <= num; j++)
        {
            cout << i << " ";
        }
        cout << endl;
    }

    return 0;
}

File: data/c#/pattern_4/code.cs
using System;
class Grammind
{
    static void Main()
    {
        int num;
        Console.Write("Enter the number of rows and columns for the square: ");
        num = Int32.Parse(Console.ReadLine());

        for (int i = num; i >= 1; i--)
        {
            for (int j = 1; j <= num; j++)
            {
                Console.Write(i + " ");
            }
            Console.Write("\n");
        }
    }
}
